nemo/collections/nlp/data/language_modeling/megatron/gpt_sft_chat_dataset.py

```diff
@@ -96,6 +116,26 @@ def _add_speaker_and_signal(header, source, mask_role, gtype):
                 + sentence["value"]
                 + END_SIGNAL
             )
+        elif gtype == "VALUE_TO_TEXT":
+            sentence["value"] = (
+                BEGIN_SIGNAL
+                + role_token
+                + sentence_from
+                + END_NAME_SIGNAL
+                + (response_value_formater(sentence['label']) if 'label' in sentence else '')
+                + sentence["value"]
+                + END_SIGNAL
+            )
+        elif gtype == "TEXT_TO_VALUE":
+            sentence["value"] = (
+                BEGIN_SIGNAL
+                + role_token
+                + sentence_from
+                + END_NAME_SIGNAL
+                + sentence["value"]
+                + END_SIGNAL
+                + (response_value_formater(sentence['label']) if 'label' in sentence else '')
+            )
         else:
             raise ValueError(f"source type {gtype} not supported")
```

Comment:
Maybe put here as the list of gtype accepted?

Reply:
done